docs/blog/2019-04-01-setting-up-yarn-workspaces-for-theme-development/index.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Setting up Yarn Workspaces for Theme Development
+date: 2019-04-01
+author: Brent Jackson
+excerpt: ""
+tags:
+  - themes
+  - yarn
+  - tutorials
+---
+
+Gatsby [themes][themes-tag] are a new way to share functionality across multiple Gatsby sites.
+Using [Yarn workspaces][] is an excellent way to set up a project for theme development.
+It allows you to keep multiple packages in a single parent directory and link dependencies together.
```

Comment:
```suggestion
[Yarn workspaces][] are an excellent way to set up a project for theme development because they allow you to keep multiple packages in a single parent directory and link dependencies together.
```
This and the previous sentence might read a little better if combined.